### PR TITLE
Adds RSN 38 for 24.02 MG cugraph `degree()` bug workaround.

### PR DESCRIPTION
**Pull request by rlratzel**
Adds RSN 38 for 24.02 MG cugraph `degree()` bug workaround.

### CODE REVIEW

File: _notices/rsn0038.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+result_df["degree"] //= 2
+```
+
+## Impact
+
+Impact is limited to RAPIDS 24.02 `cugraph` users using the graph `degree` functions (`G.degree()`, `G.in_degree()`, `G.out_degree()`) on undirected, multi-GPU distributed graphs.
+
+## Background
+
+A bug was noticed late in the 24.02 release process for `cugraph` users using the `degrees` functions (`G.degree()`, `G.in_degree()`, `G.out_degree()`) on undirected multi-GPU distributed graphs.  The bug causes the returned values to be doubled, and users of these functions for these graph types must apply a workaround to divide the result by two in order to obtain the correct values as seen in the example above.
```
Comment (bdice) [inline _notices/rsn0038.md:50]:
```suggestion
A bug was noticed late in the 24.02 release process for `cugraph` users using `degree` functions (`G.degree()`, `G.in_degree()`, `G.out_degree()`) on undirected multi-GPU distributed graphs.  The bug causes the returned values to be doubled, and users of these functions for these graph types must apply a workaround to divide the result by two in order to obtain the correct values as seen in the example above.
```

**Review by bdice — CHANGES_REQUESTED** [review]
This generally feels like the kind of thing that would warrant a hotfix rather than a notice. Users of cuGraph probably do not look at RSNs for how to handle numerical bugs. RSNs are generally reserved for changes in platform support or package/dependency incompatibility.

Are we sure we don't want to hotfix this, rather than issue an RSN?

File: _notices/rsn0038.md (hunk 1 above)
Comment (rlratzel) [reply _notices/rsn0038.md]: Thanks! I fixed this in the latest commit.

**Comment by rlratzel** [comment]
Closing.  We'll backport the fix which addresses the issue described in this RSN to PB 2.1

**Pull request closed without merging**